docs/release-notes/8.0.34-26.md

```diff
@@ -0,0 +1,150 @@
+
+      * Changing an existing value using `SET` syntax during a MySQL client session writes a warning message to the client log.
+
+      * Persisting a variable using `SET PERSIST` syntax during a MySQL client session writes a warning message to the client log.
+
+* The `keyring_file` and `keyring_encrypted_file` plugins are deprecated. These keyring plugins are replaced with the `component_keyring_file` and `component_keyring_encrypted_file` components.
+
+Find the full list of bug fixes and changes in the [MySQL 8.0.34 Release Notes](https://dev.mysql.com/doc/relnotes/mysql/8.0/en/news-8-0-34.html).
+
+## New features
+
+* [PS-8042](https://jira.percona.com/browse/PS-8042): Implements the [Audit Log Filter plugin](../audit-log-filter-overview.md).
+
+The following features and improvements were added by imlementing the Audit Log Filter plugin feature:
+
+  * [PS-3874](https://jira.percona.com/browse/PS-3874): Adds options to the Audit Plugin to allow multiple configurations.
+
+  * [PS-5164](https://jira.percona.com/browse/PS-5164): Adds `Rows_sent` and `Rows_examined` fields to the Audit Log plugin.
+
+  * [PS-5814](https://jira.percona.com/browse/PS-5814): Allows the audit log to be rotated when MySQL starts.
+
+  * [PS-236](https://jira.percona.com/browse/PS-236): Prepared statements are now logged in the audit plugin.
+
+  * [PS-2480](https://jira.percona.com/browse/PS-2480): Adds diagnostics when audit log does synchronous writes in ASYNC mode.
+
+  * [PS-7331](https://jira.percona.com/browse/PS-7331): Sets the default value of `audit_log_rotations` options to rotate.
+
+  * [PS-5945](https://jira.percona.com/browse/PS-5945): Adds the MySQL server host name to `audit_log` details when writing to the SYSLOG.
+
+* [PS-8848](https://jira.percona.com/browse/PS-8848): Increases verbosity of the connection control plugin.
+
+## Improvements
+
+*  [PS-8728](https://jira.percona.com/browse/PS-8728): Adds the data masking component to align Percona Data Masking functionality with Oracle 8.0.33.
+
+* [PS-8624](https://jira.percona.com/browse/PS-8624): Exposes the number of TABLE objects with triggers in the Table Cache.
+
+* [PS-8742](https://jira.percona.com/browse/PS-8742): Adds the `audit_log_filter_database` system variable.
+
+## Bug fixes
+
+* [PS-8811](https://jira.percona.com/browse/PS-8811): There were linking issues with protocol buffers and macOS 12 on Azure Pipelines.
+
+* [PS-3942](https://jira.percona.com/browse/PS-3942): The MySQL ping command caused the `command_class="error"` in the audit log output.
+
+* [PS-8504](https://jira.percona.com/browse/PS-8504): Setting a large value of `audit_log_filter_buffer_size` caused the server exit. I 
+
+* [PS-8770](https://jira.percona.com/browse/PS-8770): The `INSTANT DDL` caused Percona Server 8.0.32 exit after upgrading from 5.7.41.
```

Comment:
This is a bug that was fixed by Oracle, we just inherited the fix via merging. Do we treat such items the same way?

Reply:
Moved to the release highlights.